src/ws/service/friendships_service.rs

```diff
@@ -183,85 +209,119 @@ impl FriendshipsServiceServer<SocialContext> for MyFriendshipsService {
     }
 }
 
-/// Retrieve the User Id associated with the given Authentication Token.
-///
-/// If an authentication token was provided in the request, this function gets the
-/// user id from the token and returns it as a `Result<UserId, Error>`. If no
-/// authentication token was provided, this function returns a `Unauthorized`
-/// error.
-///
-/// * `request` -
-/// * `context` -
-async fn get_user_id_from_request(
-    request: &Payload,
-    synapse: SynapseComponent,
-    users_cache: Arc<Mutex<UsersCacheComponent>>,
-) -> Result<UserId, FriendshipsServiceErrorResponse> {
-    match request.synapse_token.clone() {
-        // If an authentication token was provided, get the user id from the token
-        Some(token) => get_user_id_from_token(synapse.clone(), users_cache.clone(), &token)
-            .await
-            .map_err(|_err| -> FriendshipsServiceErrorResponse {
-                FriendshipsServiceError::InternalServerError.into()
-            }),
-        // If no authentication token was provided, return an Unauthorized error.
+async fn process_room_event(
```

Comment:
just a reminder to remove this method from this file

Reply:
Yeah, do u have any suggestion for the name of the file? 🫶🏼